### DISCUSSION TITLE
Better indication for webhooks (and link them!) in triggers view

### DISCUSSION THREAD

**Discussion by anngbaum** [supabase/supabase#21238] [Feature Requests]
Small UI request, but when I was scrolling through my [triggers](https://supabase.com/dashboard/project/_/database/triggers), I noticed that my project's webhooks were appearing there, but with the function `http_request`:
<img width="1009" alt="Screenshot 2024-02-13 at 2 06 00 PM" src="https://github.com/supabase/supabase/assets/29735378/3947756c-fbed-42c6-b088-36f37624ac01">

I think this is actually fine to show them in this view (since they are just triggers under the hood), but my one small request/suggestion would be to **link to the actual webhook** in the `function` column rather than showing `http_request` because that took me a minute of searching through my own functions to just figure out what was happening.

BONUS request would be to link all of the functions in the triggers table to their corresponding RPC in the [functions](https://supabase.com/dashboard/project/_/database/functions) view!

**Comment by saltcod** [comment]
Thanks Ann! These are great requests!
